src/swapd/runtime.rs

```diff
@@ -1593,6 +1602,9 @@ impl Runtime {
                         .unwrap_or_else(|_| Duration::from_secs(0))
                         .as_secs(),
                     public_offer: self.public_offer.clone(),
+                    local_trade_role: self.local_trade_role.clone(),
```

Comment:
Same here I think (less important as it doesn't add new lines, but I'm sure clippy has something to say on using clone on copy)